Solutions/NGINX HTTP Server/Data Connectors/Connector_NGINX_agent.json

```diff
@@ -2,8 +2,7 @@
     "id": "NGINXHTTPServer",
```

Comment:
Please update `isPreview` from `true` to `false`.
Currently, it has below code:
`"availability": {
        "status": 1,
        "isPreview": true
    },`